docs/section21/Rule21-6.md

```diff
@@ -0,0 +1,40 @@
+
+# Boiler - Rule 21-6  
+
+**Rule ID:** 21-6  
+**Rule Description:** When baseline building includes two boilers each shall stage as required by load.  
+**Rule Assertion:** B-RMR  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.3.2 Building System-Specific Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-1, 5, 7, 11, or 12
+2. B-RMR is not modeled with purchased heating.
+3. B-RMR is modeled with two boilers.
+
```

Comment:
I think we only need applicability check #3, because it already accounts for #1 and #2 above.

Reply:
Check #1 and #2 are now removed.